custom/01-be-gone/go-v0.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Compile pForth with custom code and show that this works. 
+# We assume a posix shell and system (but adaption should be easy to others).
+# Note: This is the easiest solution but ignores PForths best practices set up in pfcustom.c
+# Warning: This patches the existing source tree and might create confusion when not used on separate Git branch in case an error occurs.
+# Tested on MSYS2-Cygwin, Linux, FreeBSD (X86_64 architecture each), NetBSD (i386 architecture)
+
+# save original C sources and copy demo sources. Thus we do not need to change the make file.
+mv ../../csrc/pfcustom.c ../../csrc/pfcustom_c.original
+cp ../cf_helpers.h  ../../csrc/
+cp cf_demo1.c       ../../csrc/pfcustom.c
```

Comment:
Why overwrite the file?
Can we use the CF_SOURCES (or CUSTOM_SOURCES) in the Makefile?